.github/workflows/tests.yml

```
@@ -96,8 +96,10 @@ jobs:
         if: matrix.python-version == '3.12' && matrix.os == 'ubuntu-latest' && matrix.sphinx-version == 'dev'
         uses: codecov/codecov-action@v4
         with:
-          token: ${{ secrets.CODECOV_TOKEN }}
           fail_ci_if_error: true
+          verbose: true
+        env:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

The issue you linked did the following:

```suggestion
          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

---

that's done the trick!  Thanks for your keen 👀 @gabalafou! Once CIs are all green I'll remove `verbose` and then merge.